src/app/teacher-session.service.ts

```diff
@@ -31,6 +40,12 @@ export class TeacherSessionService {
     shareReplay(1),
   );
 
+  /**
+   * An observable of the current round's session ID and round ID.
+   * Emits null if the student is not in a session or the round is not set on the session.
+   */
+  currentRoundPath$ = new BehaviorSubject<RoundPath | null>(null);
```

Comment:
Maybe this should be in the round service?

Reply:
That would work just fine, but the current round kind of feels like a property of the session.  🤷🏻‍♂️ (That's also how `StudentRoundService` treats it.)

Reply:
I do get that but it did feel weird that it was being *set* by something in the round service.

Reply:
Oh, I suppose 😄

Reply:
I agree with Joe's point about the current round being the property of the session.